coffee_maker_navigator_2/lib/main.dart

```diff
@@ -1,68 +1,28 @@
+import 'package:coffee_maker_navigator_2/app.dart';
 import 'package:coffee_maker_navigator_2/data/di/data_di_module.dart';
-import 'package:coffee_maker_navigator_2/di/dependency_injection.dart';
-import 'package:coffee_maker_navigator_2/di/di/container_manager.dart';
-import 'package:coffee_maker_navigator_2/di/di/containers.dart';
-import 'package:coffee_maker_navigator_2/di/di/injector.dart';
+import 'package:coffee_maker_navigator_2/di/di/dependency_container_manager.dart';
+import 'package:coffee_maker_navigator_2/di/di/dependency_containers/orders_dependency_container.dart';
 import 'package:coffee_maker_navigator_2/domain/di/domain_di_module.dart';
 import 'package:coffee_maker_navigator_2/ui/di/ui_di_module.dart';
-import 'package:coffee_maker_navigator_2/ui/router/view/app_router_delegate.dart';
-import 'package:coffee_maker_navigator_2/ui/router/view_model/router_view_model.dart';
-import 'package:demo_ui_components/demo_ui_components.dart';
 import 'package:flutter/material.dart';
-import 'package:provider/provider.dart';
+
+void _registerDependencyContainerFactories() {
+  final manager = DependencyContainerManager.instance;
+  manager.registerContainerFactory((_) => OrdersDependencyContainer());
+}
+
+Future<void> _initDependencyContainerManager() async {
+  final manager = DependencyContainerManager.instance;
+  await manager.init();
+  manager.registerContainerFactory((_) => OrdersDependencyContainer());
```

Comment:
probably it should be deleted, because we have _registerDependencyContainerFactories